src/components/Popups/FailedNetworkSwitchPopup.tsx

```diff
@@ -12,26 +12,32 @@ const RowNoFlex = styled(AutoRow)`
   flex-wrap: nowrap;
 `
 
+const ColumnContainer = styled(AutoColumn)`
+  margin: 0 12px 0;
+`
+
+export const PopupAlertTriangle = styled(AlertTriangleFilled)`
+  flex-shrink: 0;
+  width: 32px;
+  height: 32px;
+  margin-right: 12px;
```

Comment:
prefer putting `gap` on the parent container to putting margin on child containers 

my general opinion is that layout should be determined by parent and not individually on the child components